sway/desktop/output.c

```diff
@@ -850,6 +850,8 @@ void handle_new_output(struct wl_listener *listener, void *data) {
 		return;
 	}
 
+	wlr_output_init_render(wlr_output, server->allocator, server->renderer);
```

Comment:
Maybe we should bail out if this fails, to avoid hitting assertions down the line.

Reply:
Sure

Reply:
Let me know if the style is OK, not sure about the cut and number of tabs